modules/store/src/store_module.ts

```diff
@@ -176,6 +177,12 @@ export class StoreModule {
             initialState: config.initialState,
           },
         },
+        {
+          provide: STORE_FEATURES,
```

Comment:
```suggestion
          provide: _STORE_FEATURES,
```

and change ` @Inject(STORE_FEATURES) private features: StoreFeature<any, any>[],` to ` @Inject(_STORE_FEATURES) private features: StoreFeature<any, any>[],` in the `StoreFeatureModule` constructor.

Reply:
Are you sure about that?
I think it will be more appropriate if the `StoreFeatureModule` will receive the `STORE_FEAUTRES` token  as it already receives the`FEATURE_REDUCERS` and not the `_FEATURE_REDUCERS.`.

Please let me know what are thinking.

Reply:
The reasoning behind is because users could use `STORE_FEATURES` to add features, while `_STORE_FEATURES` is meant for internal use only. That being said you make a valid point, all of the injected items are public.

Reply:
I see your point, I will update the pull request.